### DISCUSSION TITLE
[CLI] Multiple filters with command list

### DISCUSSION THREAD

**Discussion by sgendrot** [Noovolari/leapp#377] [Q&A]
Hi,

I try to use several filters on the command list, ex:
`leapp session list  --filter="Session Name=^mysession$" --filter="Named Profile=myprofile"`

and it's not working, I try to use a commat, repeat the param filter, etc ...

So how to use several filters ? 

Thanks

**Comment by andreacavagna01** [comment]
Hi thanks for the contribution @sgendrot. I've changed the Discussion name and added the correct label. I think @ericvilla and @urz9999 can have the correct answer for this.
